### PR TITLE
[subject pages] make proto change to allow for per capita toggle on block level

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: server/config/subject_page.proto [hunk 1]
```diff
@@ -251,6 +251,11 @@ message Block {
     DISASTER_EVENT = 1;
   }
   BlockType type = 4;
+  // If denom is set, adds a per capita toggle on the block. If per capita is
+  // toggled on, all tiles in the block will use this denom to calculate per
+  // capita.
+  string denom = 6;
+  bool startPerCapita = 7;
```
Comment (pradh) [inline server/config/subject_page.proto:258]: nit: start_per_capita ... can you default to false?

Comment (pradh) [reply server/config/subject_page.proto:258]: or better call it `start_with_denom` ... since only Count_Person is per-capita.

**Review by pradh — APPROVED** [review]
Thanks!

File: server/config/subject_page.proto (hunk 1 above)
Comment (chejennifer) [reply server/config/subject_page.proto]: done & yes, it will default to false

**Pull request merged**